network-api/networkapi/wagtailpages/pagemodels/libraries/landing_page.py

```diff
@@ -23,16 +25,24 @@ class BaseLandingPage(BasePage):
         on_delete=models.SET_NULL,
         help_text="Image that will render at the top of the page.",
     )
+    body = fields.StreamField(
```

Comment:
Just a naming nitpick, but I would rename this to `about` or something on that and add a `help_text`. Right now, it seems like this is the "meat" of the page, when it is actually a blurb at the end of it.

Reply:
Hi @jhonatan-lopes!

Though I do see what you mean, from what I understand in the ticket description and [in this google doc](https://docs.google.com/document/d/19WoZxJvisMsHnXqA4GDX3Te6t9JnqAI2NG5jE9J4LP4/edit), it appears to me that stakeholders requested a page `body` streamfield, which allows users to select a "about block":

![Screenshot 2023-06-28 at 18-32-07 Research hub models](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/87208a18-95c1-4133-aac8-ee465292c1b7)


Maybe @beccaklam can help chime in on whether this field should be named `about` or `body`?

Reply:
If that's what the specs say, that's fine :)